c2corg_ui/format/ltag.py

```diff
@@ -0,0 +1,63 @@
+"""
+LTag Extension for Python-Markdown
+====================================
+
+See https://regex101.com/r/g2Zjqn/2/ for regex tests
+
+"""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from markdown.extensions import Extension
+from markdown.blockprocessors import BlockProcessor
+from markdown.util import etree
+import re
+
+class LTagProcessor(BlockProcessor):
```

Comment:
Oops, my mistake, you're right:
https://github.com/c2corg/v6_ui/blob/master/Makefile#L82

The python linter command is ``make flake8`` https://github.com/c2corg/v6_ui/blob/master/Makefile#L81